.github/workflows/cargo_fuzz.yml

```diff
@@ -25,7 +26,7 @@ jobs:
         run: ./fuzzing_setup.sh
 
       - name: Cargo fuzz build
-        run: cargo fuzz build
+        run: cd libraries/opensk && cargo fuzz build && cd ../..
```

Comment:
I'm surprised we need `cd ../..` (same in the rest of the file). I would expect `run` to start a subshell and not share the same shell as the rest of the job.

Reply:
We don't need to, removed.